src/creature.h

```diff
@@ -408,6 +408,14 @@ class Creature : virtual public Thing
 		void setUseDefense(bool useDefense) {
 			canUseDefense = useDefense;
 		}
+		void setNoMove(bool canNotMove)
```

Comment:
`canNotMove` isn't correct, `cannot` is a single word.

by the way, I can't find `cannotMove` in creature.h like you said is there.

Reply:
I meant field `bool cannotMove = false;`

Anyway, I've changed again function naming. How would you rate changes now?